docs/events-challenges/community-events.md

```diff
@@ -0,0 +1,47 @@
+# Community Events
+
+Community events are a great way to connect with other players, showcase your creativity, and enjoy everything Theatria has to offer as a community. Events often start in Discord, and everyone is encouraged to participate or contribute their ideas.
+
+## How to Get Involved
+
+### Suggest an Event
+- **Join our Discord**: Start by joining Theatria's Discord server if you haven't already. This is where event planning begins.
+- **Share your idea**: Use the designated **#event-suggestions** channel to propose your event. Provide details such as:
+  - Event name
+  - Goals or activities
+  - Date and time (or a range of possible dates)
+  - Any resources or permissions needed
+
+### Join an Event
+- **Check the schedule**: Look for upcoming events announced in the **#events** channel on Discord.
+- **Sign up**: If sign-ups are required, follow the instructions provided in the announcement.
+- **Show up and have fun**: Be ready to participate on the Minecraft server at the scheduled time.
+
+## Hosting Your Own Event
+If you want to host an event yourself, here’s what you need to do:
+
+1. **Get approval**: Submit your event proposal in the **#event-suggestions** channel for feedback and approval by the community moderators.
+2. **Plan the event**: Make sure you have all necessary resources, permissions, and any technical requirements sorted out.
+3. **Announce it**: Once approved, moderators will help you post an official announcement in the **#events** channel.
+4. **Run your event**: Be prepared to lead your event, manage participants, and have fun with your fellow players!
+
+## Event Ideas
+Not sure what kind of event to suggest? Here are some ideas:
+
+- **Build Contests**: Show off your building skills and creativity.
```

Comment:
Build contests have always been avoided on the server due to the subjective nature of judgement. I would remove this example as we don't want to encourage people to try to suggest an event like this. As cool as I think build competitions would be.